Packs/MaintenanceUpdateNotifications/Playbooks/playbook-Create_New_Maintenance_Incident.yml

```diff
@@ -0,0 +1,308 @@
+contentitemexportablefields:
+  contentitemfields:
+    propagationLabels:
+    - all
+id: Create New Maintenance Incident
+inputs:
+- description: ""
+  key: MaintenanceDetails
+  playbookInputQuery:
+  required: true
+  value: {}
+name: Create New Maintenance Incident
+outputs: []
+quiet: true
+starttaskid: "0"
+tasks:
+  "0":
+    continueonerrortype: ""
+    id: "0"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "2"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 239d9e50-d270-4d96-84d7-edd49320cf57
+      iscommand: false
+      name: ""
+      version: -1
+      description: ''
+    taskid: 239d9e50-d270-4d96-84d7-edd49320cf57
+    timertriggers: []
+    type: start
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -100
+        }
+      }
+  "1":
+    continueonerrortype: ""
+    id: "1"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "7"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      details:
+        simple: ${inputs.MaintenanceDetails.summary}
+      emailccto:
+        simple: ${incident.emailccto}
+      emailinstance:
+        simple: ${incident.emailinstance}
+      externalendtime:
+        simple: ${inputs.MaintenanceDetails.maintenanceEndDate}
+      externalstarttime:
+        simple: ${inputs.MaintenanceDetails.maintenanceStartDate}
+      incidenturl:
+        simple: ${inputs.MaintenanceDetails.link}
+      monitoremailsubject:
+        simple: ${incident.monitoremailsubject}
+      name:
+        simple: ${inputs.MaintenanceDetails.title}
+      sendemailto:
+        simple: ${incident.sendemailto}
+      type:
+        simple: Maintenance Activity
+      updatedetails:
+        simple: ${inputs.MaintenanceDetails.summary}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Builtin
+      description: commands.local.cmd.create.inc
+      id: 490ac920-9c16-428a-8667-5252183638b7
+      iscommand: true
+      name: Create New Incident
+      script: Builtin|||createNewIncident
+      type: regular
+      version: -1
+    taskid: 490ac920-9c16-428a-8667-5252183638b7
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 740,
+          "y": 470
+        }
+      }
+  "2":
+    continueonerrortype: ""
+    id: "2"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "4"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      extend-context:
+        simple: foundIncidentID=Contents.data.id
+      fromdate:
+        simple: 30 days ago
```

Comment:
Perhaps setting this as an input for that playbook with a default value can be much more customizable for the user - WDYT?